docs/src/tutorials/RK_order_conditions.md

```diff
@@ -0,0 +1,82 @@
+# Runge-Kutta Order conditions
+
+In this tutorial, we generate order conditions for a generic explicit Runge-Kutta method.
+
+First, we create symbolic coefficient arrays with the appropriate structure.
+
+
+```julia
+using BSeries, SymPy, Latexify
+
+s = 4  # Stages
+p = 4  # Desired order of accuracy
+
+A = Array{Sym,2}(undef,s,s)
+b = Array{Sym,1}(undef,s)
+c = Array{Sym,1}(undef,s);
+```
```

Comment:
If you turn these parts into (named) example blocks (from Documenter.jl) as in
https://github.com/ranocha/BSeries.jl/blob/3a4510816975eda6b0ea39194598ad6e1cf92456/docs/src/tutorials/bseries_basics.md?plain=1#L7-L13
the code will be executed whenever we build the docs to ensure that everything still works. 


````suggestion
```@example bseries-RK-order-conditions
using BSeries, SymPy, Latexify

s = 4  # Stages
p = 4  # Desired order of accuracy

A = Array{Sym,2}(undef, s, s)
b = Array{Sym,1}(undef, s)
c = Array{Sym,1}(undef, s);
```
````